### ISSUE TITLE
"That model does not exist" error

### ISSUE THREAD

**Issue by oakif** [gragland/chatgpt-everywhere#31]
When trying to run the server, I get the following error: 

```
$ node server.js
✖ Training ChatGPT (1 plugin rules)
file:///Users/user/chatgpt-chrome-extension/node_modules/chatgpt/build/index.js:202
              const error = new ChatGPTError(msg, { cause: res });
                            ^

ChatGPTError: ChatGPT error 404: {
    "error": {
        "message": "That model does not exist",
        "type": "invalid_request_error",
        "param": null,
        "code": null
    }
}

    at file:///Users/user/chatgpt-chrome-extension/node_modules/chatgpt/build/index.js:202:29
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  statusCode: 404,
  statusText: 'Not Found',
  [cause]: Response {
    [Symbol(realm)]: null,
    [Symbol(state)]: {
      aborted: false,
      rangeRequested: false,
      timingAllowPassed: true,
      requestIncludesCredentials: true,
      type: 'default',
      status: 404,
...
```

**Comment by JustBnrh** [comment]
Same Problem

**Comment by craigSochocki** [comment]
Same here, too.  Do you all have ChatGPT Plus?

**Comment by munawarabadullah** [comment]
Yes, I have ChatGPT Plus.

**Comment by oakif** [comment]
Same here

**Comment by gragland** [comment]
@josephvtesta Any idea what the cause would be? Issue seemed to start happening after merging your pull request.

**Comment by AdamSiyala** [comment]
"npm install chatgpt" fixed it for me.

**Comment by maurop123** [comment]
> "npm install chatgpt" fixed it for me.

Worked for me too. `chatgpt v4.4.0` seems to work

**Comment by ControlledExplosion** [comment]
Same issue as a free plan user. @AdamSiyala's solution works for me as well.

**Comment by josephvtesta** [comment]
> @josephvtesta Any idea what the cause would be? Issue seemed to start happening after merging your pull request.

Yep, @AdamSiyala is correct. Make sure you've got chatgpt 4.0.0 or greater installed. All is working for me.

**Comment by ajesuscode** [comment]
I just modified on server.js  completionParams, because console.log was showing me a wrong one`const gptApi = new ChatGPTAPI({
    apiKey: process.env.OPENAI_API_KEY,
    completionParams: {
        model: "text-davinci-003",
        top_p: 0.8,
    },
    maxResponseTokens: 300,
});
console.log(gptApi);`

**Comment by MustafaAlsaegh** [comment]
I keep getting this problem 
ChatGPTError: ChatGPT error 429: {
    "error": {
        "message": "You exceeded your current quota, please check your plan and billing details.",
        "type": "insufficient_quota",
        "param": null,
        "code": null
    }

**Comment by ajesuscode** [comment]
> I keep getting this problem ChatGPTError: ChatGPT error 429: { "error": { "message": "You exceeded your current quota, please check your plan and billing details.", "type": "insufficient_quota", "param": null, "code": null }

It says that you used all your free api quota from provided api_key
Please log in into your account and click Usage. There you will see Free Trial Usage.
If you have 0 or you spent all 18 dollars grant it means you need new account and new free trial api key
